app/regen/cmd/root.go

```diff
@@ -72,7 +72,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 				return err
 			}
 
-			return server.InterceptConfigsPreRunHandler(cmd)
+			return server.InterceptConfigsPreRunHandler(cmd, "", nil)
```

Comment:
This is actually an undocumented breaking change between beta1 and rc0. I created https://github.com/cosmos/cosmos-sdk/pull/9592